Description: Drop unpackaged happybase from dependencies
 This avoids breaking stevedore extensions via endpoints.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

--- a/tools/pip-requires
+++ b/tools/pip-requires
@@ -25,4 +25,4 @@ extras
 wsme>=0.5b1
 pyyaml
 http://tarballs.openstack.org/oslo-config/oslo.config-1.1.0b1.tar.gz#egg=oslo.config
-happybase>=0.4
+#happybase>=0.4
